quesma/queryparser/pancake_json_rendering.go

```diff
@@ -40,18 +40,18 @@ func (p *pancakeJSONRenderer) selectMetricRows(metricName string, rows []model.Q
 	return
 }
 
-func (p *pancakeJSONRenderer) selectTopHitsRows(topHits *pancakeModelMetricAggregation, rows []model.QueryResultRow) (result []model.QueryResultRow) {
+func (p *pancakeJSONRenderer) selectTopRows(topAggr *pancakeModelMetricAggregation, rows []model.QueryResultRow) (result []model.QueryResultRow) {
```

Comment:
I think comment is needed what this function does. With `TopHits` in name it was more or less clear, now I don't think so.

Reply:
Will keep TopHits convention and add comment.

The re-name column logic could be simplified once we move completely to pancakes.